### DISCUSSION TITLE
Add Namespace to helper class file

### DISCUSSION THREAD

**Discussion by kmd-muc** [joomengine/Joomla-Component-Builder#825] [Q&A] [answered]
Hi,
in want to add an additional namespace to my components helper class file.
At the beginning of the file, there already are some namespaces:
use Joomla\CMS\Filesystem\File;
use Joomla\CMS\Language\Language;
use Joomla\Registry\Registry;
use Joomla\String\StringHelper;
use Joomla\Utilities\ArrayHelper;
/***Add another one here like this:***/
use AcyMailing\Helpers\MailerHelper;

How would I add an additional namespace to this block without the compiler overwriting the changes?

**Comment by ohrionmartin** [comment, marked as answer]
Hi @kommid,

Please watch this [video](https://youtu.be/TjoH5rj4UgM), it will show you how to achieve that. This [video](https://youtu.be/sdKrX7y1mHs) will show you how to get access to the plugin, and here is the [link](https://git.vdm.dev/joomla/plg_content_componentbuildercomponentheaderstabs/releases/tag/v1.0.0) to the plugin that will allow you to achieve your goal. **Please note that you must first create an account in order to get access to the plugins.**

Hope this helps, cheers!

**Reply by kmd-muc** [reply]
Thank you very much!
